Source/Fuse.iOS/iOSApp.uno

```diff
@@ -126,7 +141,7 @@ namespace Fuse
  			if (_prevStatusBarOrientation != o)
  			{
  				_prevStatusBarOrientation = o;
-				UpdateManager.PerformNextFrame(_graphicsView.InvalidateVisual);
+				UpdateManager.PerformNextFrame(RootVisual.InvalidateVisual);
```

Comment:
No, but I changed to code to make it clearer https://github.com/fusetools/fuselibs-public/pull/31/commits/47bc5bf8444de9db6396d071e35f94af1f003249#diff-4146b1a7d97819d8c81051849608276fR144